Dockerfile

```diff
@@ -1,4 +1,30 @@
+#-----------------------------------------------------#
+#                     System Base                     #
+#-----------------------------------------------------#
 FROM openjdk:8u171-alpine3.7
 RUN apk --no-cache add curl
-COPY target/oncostore*.jar oncostore.jar
-CMD java ${JAVA_OPTS} -jar oncostore.jar
\ No newline at end of file
+
+#-----------------------------------------------------#
+#                Install Variantstore                 #
+#-----------------------------------------------------#
+# Define install directory
+ENV variantstore=/root/variantstore
+RUN mkdir ${variantstore}
+
+# Get required files for setup
+COPY ./src ${variantstore}/src
+COPY ./pom.xml ${variantstore}/pom.xml
+COPY ./micronaut-cli.yml ${variantstore}/micronaut-cli.yml
+COPY ./.mvn ${variantstore}/.mvn
+COPY ./mvnw ${variantstore}/mvnw
+COPY ./mvnw.cmd ${variantstore}/mvnw.cmd
+
+# Compile Variantstore
+RUN cd ${variantstore} && \
```

Comment:
We could also use `mvn clean package` here, what do you think?

Reply:
And then running the resulting jar, instead of mvn exec:exec?
Sure!

Reply:
Exactly, that would be the plan. 👍

Reply:
Linked to #47